<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibManageWindow</class>
 <widget class="QMainWindow" name="LibManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SaveFileButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存更改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正在编辑：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileDisplayLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="ParapListWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="WordListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddParapButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加词义</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditParapButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改词义</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeletParapButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除词义</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddWordButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加单词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeletWordButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除单词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditWordButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看/编辑</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CharComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>n.   </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pron.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>adj. </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>adv. </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>v.   </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>num. </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>art. </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prep.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>conj.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>int. </string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ParapLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
